src/main/scala/intellij/haskell/action/DownloadCabalConfigFile.scala

```diff
@@ -0,0 +1,17 @@
+package intellij.haskell.action
+
+import com.intellij.openapi.actionSystem.{AnAction, AnActionEvent}
+import intellij.haskell.external.component.HaskellComponentsManager.refreshGlobalProjectInfo
+import intellij.haskell.external.component.PathComponent
+
```

Comment:
Ok, fair enough. I will remove the action then 😄. Users should not worry about some internal relations too much indeed.